sonic-xcvrd/scripts/xcvrd

```diff
@@ -183,21 +204,42 @@ def post_port_dom_info_to_db(logical_port_name, table, stop=threading.Event()):
             dom_info_dict = platform_sfputil.get_transceiver_dom_info_dict(physical_port)
             if dom_info_dict is not None:
                 beautify_dom_info_dict(dom_info_dict)
-                fvs = swsscommon.FieldValuePairs([('temperature', dom_info_dict['temperature']),
-                                                  ('voltage', dom_info_dict['voltage']),
-                                                  ('rx1power', dom_info_dict['rx1power']),
-                                                  ('rx2power', dom_info_dict['rx2power']),
-                                                  ('rx3power', dom_info_dict['rx3power']),
-                                                  ('rx4power', dom_info_dict['rx4power']),
-                                                  ('tx1bias', dom_info_dict['tx1bias']),
-                                                  ('tx2bias', dom_info_dict['tx2bias']),
-                                                  ('tx3bias', dom_info_dict['tx3bias']),
-                                                  ('tx4bias', dom_info_dict['tx4bias']),
-                                                  ('tx1power', dom_info_dict['tx1power']),
-                                                  ('tx2power', dom_info_dict['tx2power']),
-                                                  ('tx3power', dom_info_dict['tx3power']),
-                                                  ('tx4power', dom_info_dict['tx4power'])])
+                beautify_dom_threshold_info_dict(dom_info_dict)
+                fvs = swsscommon.FieldValuePairs(
+                        [('temperature', dom_info_dict['temperature']),
+                         ('voltage', dom_info_dict['voltage']),
+                         ('rx1power', dom_info_dict['rx1power']),
+                         ('rx2power', dom_info_dict['rx2power']),
+                         ('rx3power', dom_info_dict['rx3power']),
+                         ('rx4power', dom_info_dict['rx4power']),
+                         ('tx1bias', dom_info_dict['tx1bias']),
+                         ('tx2bias', dom_info_dict['tx2bias']),
+                         ('tx3bias', dom_info_dict['tx3bias']),
+                         ('tx4bias', dom_info_dict['tx4bias']),
+                         ('tx1power', dom_info_dict['tx1power']),
+                         ('tx2power', dom_info_dict['tx2power']),
+                         ('tx3power', dom_info_dict['tx3power']),
+                         ('tx4power', dom_info_dict['tx4power']),
+                         ('temphighalarm', dom_info_dict['temphighalarm']),
```

Comment:
I would suggest separating below to entries into two parts,  for the constant entries, we only fetch them one time at the starting of the xcvrd as port_info, for the variable ones fetch them periodically with other DOM entries.

Reply:
Hi,
Implemented  a new function post_port_dom_threshold_info_to_db. This function will be called only during init, New Optic Detection.  Dom Info will be called periodically every 1 minute via dom_info_update_task . We are not calling the dom_threshold update here

Thanks
Sridhar.R

Reply:
is it still reading the threshold value every time when updating the dom info?  only not updated to DB? if yes I would like to suggest not to do it because reading  EEPROM is time-consuming, don't want to introduce unnecessarily load.

Reply:
> is it still reading the threshold value every time when updating the dom info? only not updated to DB? if yes I would like to suggest not to do it because reading EEPROM is time-consuming, don't want to introduce unnecessarily load.

I think this comment is still valid, I suggest to decouple the threshold reading and voltage/power/temp/bias reading, please refer to my comments in another PR.

Reply:
Hi @keboliu,

Implemented a new function get_transceiver_dom_threshold_info_dict. This will read only dom threshold.
Removed reading threshold values in existing get_transceiver_dom_info_dict.

Hope this time i have not mis understood your comments.